mowgli_etl/pipeline/wdc/parsimonious_parser/wdc_parsimonious_node_visitor.py

```diff
@@ -17,10 +36,12 @@ def visit_dimension(self, node, visited_children):
             key = "width"
         elif key == "h":
             key = "height"
-        self.dictionary[key] = float(value)
+        self.dictionary.__getattribute__(key).value = float(value)
```

Comment:
getattr(self.dictionary, key). You shouldn't ever need to call magic methods directly.